helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -1242,6 +1242,44 @@ certmanager:
   enabled: false
   installCRDs: true
 
+################################################################################
+# Target Allocator
+# Notice: Target Allocator related features should be considered to have an alpha
+# maturity level and be experimental. There may be breaking changes or Operator
+# features may be replaced entirely with a better alternative in the future.
+#
+# The Target Allocator is running as a deployment with a replica count of 1.
+# It discovers scraping configurations from ServiceMonitor and PodMonitor CRDs and
+# assigns them to collectors.
+# Related documentation: https://github.com/open-telemetry/opentelemetry-operator/tree/main/cmd/otel-allocator
+################################################################################
```

Comment:
We should also mention that it conflicts with `autodetect.prometheus`. Only one of them should be enabled.

Reply:
It doesn't conflict ; these seem completely independent.

Reply:
Do you mean that annotated endpoints are never scraped with the target allocator? By conflict, I mean collecting the same metrics twice.